### DISCUSSION TITLE
A sandbox save UI popped up even if there's no mods intalled

### DISCUSSION THREAD

**Discussion by Zeppelin5079** [trumank/mint#222] [Q&A] [answered]
I used to use ingame mod system to install mod from mod.io until I found this treasure tool.

However, everytime I install mods using mint, a pop up UI asking me to create a sandbox save will appear. And if I cancel the UI by right clicking, the game will just freeze.

![Untitled](https://github.com/trumank/mint/assets/57742246/8a8033cc-e42b-4892-b780-5849b4d56cca)

After that I try to uninstall all the mods leaving mint with blank mod_P.pak, the pop up still appears.
![Untitled](https://github.com/trumank/mint/assets/57742246/d13f406e-d42e-49e0-91f2-19f3c827056d)

In short, it seems like that if there's a mod_P.pak installed, the game will detected it as sandbox mod. 

Therefore I need some help, thanks.

**Comment by jieyouxu** [comment, marked as answer]
Possibly your antivirus flagged and removed the .dll hook after it was written, maybe https://github.com/trumank/mint/issues/175?

**Reply by Zeppelin5079** [reply]
Thanks for helping!

It did turns out to be my AV's problem, not even a single notification of the file being blocked was given. lol
